legal-api/migrations/versions/ec9652171563_review.py

```diff
@@ -0,0 +1,56 @@
+"""review
+
+Revision ID: ec9652171563
+Revises: 01b28a2bb730
+Create Date: 2024-06-26 16:13:07.413995
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = 'ec9652171563'
+down_revision = '01b28a2bb730'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column('filings', sa.Column('resubmission_date', sa.DateTime(timezone=True), nullable=True))
```

Comment:
This is the "last resubmission date", right? (In case the filing was resubmitted multiple times.)

Reply:
yes